covid_xprize/examples/prescriptors/neat/README.md

```diff
@@ -50,26 +49,45 @@ The script configures neat-python using the file `config-prescriptor`.
 This file contains many options for the underlying algorithm.
 
 
-
 ### Prescribing with trained Prescriptors
 
 The script `prescribe.py` implements the API required for prescriptor submissions.
 
-As is, it simply loads one of the checkpoints from training,
-and makes prescriptions with each network in that checkpoint.
+As is, it simply loads one of the checkpoints from training, and makes prescriptions with each network 
+in that checkpoint.
 
 Say we would like to prescribe using the checkpoint `neat-checkpoint-4`.
 Then, open `prescribe.py` and change the line defining `PRESCRIPTORS_FILE` to
-```
+```shell script
 PRESCRIPTORS_FILE = 'neat-checkpoint-4'
 ```
 
 Then, prescriptions can be generated by following the API.
 For example, say we would like to generate predictions from August 1st to 5th,
-and store the results in `test_prescriptions.csv`:
+and store the results in `predictions/test_prescriptions.csv`. 
+Run this from the root of your cloned repository:
+```shell script
+python covid_xprize/examples/prescriptors/neat/prescribe.py \
+    --start_date 2020-08-01 \
+    --end_date 2020-08-05 \
+    --interventions_past ./covid_xprize/validation/data/2020-09-30_historical_ip.csv \
+    --intervention_costs ./covid_xprize/validation/data/uniform_random_costs.csv \
+    --output_file predictions/test_prescriptions.csv
 ```
-cd <workspace>/covid-xprize/covid_xprize/examples/prescriptors/neat/
-python prescribe.py --start_date 2020-08-01 --end_date 2020-08-05 -ip ../../../validation/data/2020-09-30_historical_ip.csv -c ../../../validation/data/uniform_random_costs.csv -o test_prescriptions.csv
+
+You can validate the prescriptions generated in the above step using the validator module as follows:
```

Comment:
@ofrancon I thought it might be useful for teams to experience the validation step manually, and also as a confirmation that everything went as expected with the NEAT predictor sample.